content/en/_index.md

```diff
@@ -983,6 +983,11 @@ Examples of portable practices include:
 Code snippets should have reasonable test coverage and all critical code paths
 should have integration tests that test against the production service.
 
+### 1-to-1 test cases to snippets
+
+Each individual sample method or snippet should have a single dedicated test
+case (test method or function).
```

Comment:
And the test case should be in its own file, correct?

Reply:
Ideally, yes. Golang has a lot of 1-file-to-many-test-cases situations, so I felt a little hypocritical recommending that.

Reply:
I'm not sure I'm following the guideline being proposed.

It seems like we want 1+ test cases for each sample?

Is this a guideline about avoiding a single test checking multiple samples?

Reply:
Yes, exactly. We want to avoid having a single test function that checks multiple samples.

See anti-pattern here: https://github.com/GoogleCloudPlatform/golang-samples/blob/main/bigtable/writes/writes_test.go